.github/workflows/projects_check.yml

```diff
@@ -8,18 +8,29 @@ on:
     paths:
     - '.github/workflows/xcode_check.yml'
     - 'EndlessSky.xcodeproj/**'
+    - 'EndlessSkyLib.cbp'
+    - 'EndlessSkyTests.cbp'
     - 'source/**'
     - 'tests/check_xcode.sh'
+    - 'tests/check_codeblocks.sh'
   pull_request:
     types: [opened, synchronize]
     paths:
     - '.github/workflows/xcode_check.yml'
     - 'EndlessSky.xcodeproj/**'
+    - 'EndlessSkyLib.cbp'
+    - 'EndlessSkyTests.cbp'
     - 'source/**'
     - 'tests/check_xcode.sh'
+    - 'tests/check_codeblocks.sh'
 
 jobs:
-  update-xcode-project:
+  changed:
+    uses: ./.github/workflows/compute-changes.yml
+
+  check-xcode:
+    needs: changed
+    if: ${{ needs.changed.outputs.xcode_files }}
```

Comment:
These jobs need to also run when game code is modified, or they won't catch missing updates